docs/js/wave.js

```diff
@@ -1,9 +1,10 @@
 class Wave {
-    current_stream = {};
-    sources = {};
-    onFileLoad;
 
-    constructor() { }
+    constructor() {
+        this.current_stream = {};
+        this.sources = {};
+        this.onFileLoad;
```

Comment:
Is there a need to declare it since it's no value is assigned?

Reply:
Empty objects are still values assigned.. I have tested without, which cause a few errors.. So better leave them be :P